data/fontdata.json

```diff
@@ -1,21 +1,25 @@
 {
   "//1": "If more than one font is specified for a typeface the list is treated as a fallback order.",
   "//2": "unifont will always be used as a 'last resort' fallback even if not listed here.",
+  "//3": "Fonts can be provided as either a string or an object. The object must have a path",
+  "//4": "attribute, which provides the path to the font. It may also have a 'hinting'",
+  "//5": "attribute, which determines the font hinting mode. Otherwise, 'default' is used.",
+  "//6": "hinting may be one of: Auto, NoAuto, Default, Light, or Bitmap",
```

Comment:
Given the size of the comment now, would it be a good idea to move this out into a dedicated file somewhere in the `docs/` (and leave a pointer to said file here)?

Relatedly, this comment is only present in the "internal" `data/fontdata.json` but not in the user-facing (and supposed-to-be-overridable) `config/fonts.json`. I'm not quite sure why, and this is technically really out of scope of your PR, but it would be cool if `config/fonts.json` had a pointer to documentation...

(I don't have any comment of substance for the actual code)

Reply:
Documentation is now in `docs/FONT_OPTIONS.md` and is linked from both `config/fonts.json` and `data/fontdata.json`.